node/impl/full/eth_trace.go

```diff
@@ -498,6 +499,13 @@ func traceEthCreate(env *environment, addr []int, et *types.ExecutionTrace) (*et
 		// So we don't try and include a sentinel "impossible bytecode"
 		// value (the value specified by EIP-3541).
 		output = []byte{0xFE}
+		if deployedCode != nil {
+			var getBytecodeReturn evm12.GetBytecodeReturn
+			if err := getBytecodeReturn.UnmarshalCBOR(bytes.NewReader(et.MsgRct.Return)); err == nil && getBytecodeReturn.Cid != nil {
+				deployedCode[abi.ActorID(id)] = getBytecodeReturn.Cid
```

Comment:
@Stebalien This is not correct. How do I get the byte code of the contract here ?

Reply:
Do I need to make a call to the `GetBytecode` method of the contract Actor ?
Looks like none of the trace types we here have the byte code in them.

Reply:
Sorry, I missed this question. You can actually just read the bytecode from the EVM actor state (load the actor and call `GetBytecode` using the actor shims.

However, you have to do this _after_ building the trace (see https://github.com/filecoin-project/lotus/pull/11984#discussion_r1612060558)